src/modules/Builder/features/SaveAndPublish/SaveAndPublish.tsx

```diff
@@ -41,12 +41,12 @@ export const SaveAndPublish = () => {
 
     builderSessionStorage.removeItem();
 
-    if (!isNewApplet) return;
-
-    if (createApplet.fulfilled.match(result)) {
+    if (isNewApplet && createApplet.fulfilled.match(result)) {
```

Comment:
if this changes have refactoring purpose only, please revert, I have changes in this file in my PR